source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -38,9 +38,15 @@ The following attributes were added :
 
 Following enums were defined :
 - InternalReference
-- NIDCPOWER_VAL_CANCEL
-- NIDCPOWER_VAL_COMMIT
+- CalibrationCloseAction
```

Comment:
note: Line 41 is just fixing the change documentation. Instead of listing individual values from enum, we are now listing name of the enum itself.

Reply:
Correct. So can I resolve this comment?

Reply:
Yes, I just added it as note for owners so that they know what this change is.